src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -212,6 +212,9 @@
 Cluster_Diagnostic_Checks_Job_Log_Save_Failed = 'Failed to save cluster diagnostic checks job log'
 # Diagnostic Results Name
 Outbound_Connectivity_Check_Result_String = "Outbound Network Connectivity Result:"
```

Comment:
I think this is a little confusing. Let's name the regular test as "Outbound_Connectivity_Check_For_Onboarding" and "Outbound_Connectivity_Check_For_ClusterConnect" to make it clear in the code as well

Reply:
Same comment for logs as well which we're appending in the arc_diagnostic_logs folder

Reply:
Yeah but OBO is not the only endpoint for cluster connect right , so if any other endpoint required for cluster connect fails then the log will not give the right impression to us and to the user.